bazel/http-server/server.ts

```diff
@@ -174,9 +175,10 @@ export class HttpServer {
   }
 
   private _corsMiddleware(req: http.IncomingMessage, res: http.ServerResponse) {
-    res.setHeader('Access-Control-Allow-Origin', '*');
-    res.setHeader('Cross-Origin-Embedder-Policy', 'require-corp');
-    res.setHeader('cross-origin-opener-policy', 'same-origin');
+    if (this._relaxCors) {
```

Comment:
Any specific reason for dropping the wildcard for `Access-Control-Allow-Origin` ?

Reply:
I put it back.